content/pricing/faqs.textile

```diff
@@ -159,8 +159,29 @@ Yes.
 
 Upgrades take effect immediately, whilst downgrades will take effect at the beginning of the following month.
 
-h3(#cancel). Can I cancel my account at any time?
-
-Yes. If you wish to cancel your account, downgrade to the "Free package":/docs/pricing/free first and then have the account owner "contact us":https://ably.com/support. Be aware that this will permanently delete all the information associated with your account.
+h3(#cancel). Can I close my account at any time?
+
+Yes. You can close your account at any time by downgrading to the "Free package":/docs/pricing/free. The following steps outline the the process to close your account:
+
+* Ensure you are the account "owner":/docs/account/users#roles.
+* Downgrade all paid "packages":/docs/pricing#packages free packages:
+** Navigate to *Account* then *Billing*.
+** Click *Save billing details* to save your changes.
+* After all packages have been downgraded to *Free*, go to the *My Settings* page.
+** Navigate to *Want to delete your profile?*.
+** Click *start* to begin the closure process.
+* On the *Close Your Ably Account* page:
+** Review the account summary, including ownership, plan type, and app details.
+** Confirm all accounts are on the *Free* tier and details are accurate.
+** Click *Confirm Closure* to permanently deactivate your Ably account.
+
+If you sign in with SSO, for example, Google or GitHub, you cannot use the self-service account closure process. To proceed, you must set a password and disconnect their SSO provider via My Settings.
+
+* Log in to your Ably account using your current SSO method (Google or GitHub).
+* Navigate to *Account* then *My Settings*.
+* In the *Password* section, click *Change your password*.
+* Click *Update my personal settings* to save the changes.
+* Scroll to the *Login provider* section.
+* Click *remove connection* next to the SSO provider/s you want to disconnect.
```

Comment:
I'd link out to the section you've written above rather than repeating all these steps in the FAQ.